src/main/java/uk/gov/companieshouse/account/validator/service/maintenance/AccountMaintenanceService.java

```diff
@@ -29,7 +24,7 @@ public class AccountMaintenanceService {
     private final RequestStatusRepository statusRepository;
 
     @Value("${delete.files.older.than.days}")
-    private int DAYS_TO_DELETE;
```

Comment:
This was changed to match standard CamelCase styling since this is not a `const`